static/js/chart/draw_choropleth.ts

```diff
@@ -37,6 +38,28 @@ const LEGEND_IMG_WIDTH = 10;
 const NUM_TICKS = 5;
 const REDIRECT_BASE_URL = `/place/`;
 
+/**
+ * From https://bl.ocks.org/HarryStevens/0e440b73fbd88df7c6538417481c9065
+ * scales and translates the projection to allow resizing of the choropleth map
+ */
+function fitSize(
+  width: number,
+  height: number,
+  object: any,
+  projection: any,
```

Comment:
could we try to get the type for this (since we are calling operations on it)
for object as well, since we have the geojson type now